plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsViewModel.java

```diff
@@ -128,6 +135,10 @@ protected void updateSourcePath(String sourcePath, org.eclipse.core.resources.IP
 	protected void updatePodPath(String newPodPath, Map<String, IDeploymentResourceMapper> deploymentResourceMapperByProjectName, IService service) {
```

Comment:
That is how it worked before. I did not touch that logic as it does not relate to editor section. I only provided that editor section might put through its value got from server without being ignored in this way.